lib/byebug/helpers/toggle.rb

```diff
@@ -26,14 +24,22 @@ def enable_disable_breakpoints(is_enable, args)
           selected_breakpoints = all_breakpoints.select do |b|
             selected_ids.include?(b.id)
           end
+          return selected_breakpoints
```

Comment:
> You took more time replying to my comments than it would have taken you to address them. 

Actually that's not true, if you count the time to _test_ the code and update git. As you know this code was written a while back in an environment when I had byebug set up. I often don't have  that environment as easily accessible. 

Also, the same could be said of you. The time you spend writing the  critique could have been spend just adjusting the code. But the difference is that you'ld adjust it in a way that was pleasing to you and there wouldn't be back and forth. It's not clear that on the second or third try this would be the case for me. 

So, yes, thanks for taking this on and finishing it in a way you find pleasing.